source/features/forked-to.tsx

```diff
@@ -57,19 +59,49 @@ async function init(): Promise<void> {
 
 	const forks = await cache.get<string[]>(getCacheKey());
 
-	if (forks) {
-		const pageHeader = select('.pagehead h1.public')!;
-		for (const fork of forks.filter(fork => fork !== getRepoURL())) {
-			pageHeader.append(
-				<span className="fork-flag rgh-forked">
-					forked to <a href={`/${fork}`}>{fork}</a>
-				</span>
-			);
-		}
+	if (!forks) {
+		return;
+	}
+
+	document.body.classList.add('rgh-forked-to');
+
+	const forkCounter = select('.social-count[href$="/network/members"]')!;
+	if (forks.length === 1) {
+		forkCounter.before(
+			<a href={`/${forks[0]}`}
+				className="btn btn-sm float-left rgh-forked-button"
+				title={`Open your fork to ${forks[0]}`}>
+				{icons.externalLink()}
+			</a>
+		);
+	} else {
+		forkCounter.before(
+			<details className="details-reset details-overlay select-menu float-left">
+				<summary
+					className="select-menu-button float-left btn btn-sm btn-with-count rgh-forked-button"
+					title="Open any of your forks"/>
+				<details-menu
+					style={{zIndex: 99}}
+					className="select-menu-modal position-absolute right-0 mt-5">
+					<div className="select-menu-header">
+						<span className="select-menu-title">Your forks</span>
+					</div>
+					{...forks.map(fork =>
+						<a
+							href={`/${fork}`}
+							className="select-menu-item"
+							title={`Open your fork to ${fork}`}>
+							{icons.fork()}
+							{fork}
+						</a>
+					)}
+				</details-menu>
+			</details>
+		);
 	}
 
 	// Validate cache after showing links once, to make it faster
-	await updateForks(forks || []);
+	await updateForks(forks);
 }
 
 features.add({
```

Comment:
Before merging:

- Update screenshot here and in readme
- Update the description here and in readme (and include trailing period). Probably "Adds a shortcut to your forks next to the `Fork` button on the current repo."

Possible future changes:

- `await updateForks(forks)` should probably be replaced by that code that triggers the lightbox to load, so our existing `load` handler can check all the forks at once. `updateForks` and `validateFork` can then be dropped altogether.
- I think #2334 hasn't been by this PR